docs/06_how-to-guides/30_key-value-api/kv_table/20_how-to-create-indexes-kv-table.md

```diff
@@ -107,15 +108,16 @@ class [[eosio::contract]] smrtcontract : public contract {
 };
 ```
 
-### Define a non-unique index on property first_name using the macro KV_NAMED_INDEX
+### Define a non-unique index on the data member first_name using the macro KV_NAMED_INDEX
 
 1. Use the `KV_NAMED_INDEX` with two parameters.
-2. Pass the name of the index as the first parameter. The parameter must be a qualified `eosio::name`. See documentation for the [eosio name restrictions](https://developers.eos.io/welcome/latest/glossary/index#account-name).
-3. Pass the name of the property for which the index is defined as the second parameter.
+2. The first parameter is the name of the index and must be a qualified `eosio::name`. See documentation for the [eosio name restrictions](https://developers.eos.io/welcome/latest/glossary/index#account-name).
+3. The second parameter is the name of the data member the index is defined for.
+4. Call the base class method `init()` in the constructor of `address_table` type and pass the contract name as the first parameter and `first_name` index, by the KV_NAMED_INDEX macro, as the second parameter.
```

Comment:
applies to all these:

 and `first_name` index, by the KV_NAMED_INDEX macro, as the second parameter.

The line above is not clear + try to match it to the previous edits where:

The first parameter is ...
The second parameter is ....

Reply:
reworked